### ISSUE TITLE
Coin collection retained on both Level and Start Reset

### ISSUE THREAD

**Issue by Malkierian** [aglab2/hacktice#37]
I don't know which one is supposed to reset coin collection, but one of them should, otherwise at the moment I don't see a difference between the two. Note that this is only tested on the vanilla game and vanilla+ hacks (like Super Captain Falcon 64).

**Comment by aglab2** [comment]
Hi. Is that happening on latest 1.6.2 version of hacktice? I have recently changed the logic so there could be a regression there.

**Comment by Malkierian** [comment]
Yes, and it's been that way since 1.4.3, which is what I just updated from.
